src/openrct2/scenario/Scenario.cpp

```diff
@@ -66,8 +68,7 @@ uint16_t gSavedAge;
 uint32_t gLastAutoSaveUpdate = 0;
 
 uint32_t gScenarioTicks;
-uint32_t gScenarioSrand0;
-uint32_t gScenarioSrand1;
+static Random::Rct2Engine rnd;
```

Comment:
I would rather call this gScenarioRand

Reply:
Sure, I've exposed this `gScenarioRand` in `Scenario.hpp` as well.